lib/Corres_Method.thy

```diff
@@ -5,22 +5,9 @@
  *)
 
 theory Corres_Method
-imports Corres_UL SpecValid_R
+imports Corres_Cases SpecValid_R
 begin
 
-(*TODO move this *)
-
-method_setup repeat_new =
```

Comment:
this got removed in this commit, but I've not seen it moved... does that mean we have an existing `repeat_new` this was shadowing?

Reply:
yes, apparently `repeat_new` was added ages ago already and the move in here was forgotten/never noticed